### PR TITLE
Bisrat | about page

### PR DESCRIPTION
**Pull request by bisrat-walle**
image
image
image
image
image

### CODE REVIEW

File: Web/starter_project_vanguard/pages/about/index.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+          </p>
+        </div>
+        <Activities
+          studentGrowthRate={180}
+          learningRate={100}
+          growthRate={20}
+        />
+        <div>
+          <p className="font-extrabold text-[#264FAD] text-4xl my-10">
+            <span className="font-lato text-[#2B2A35]">The Problem We </span>{' '}
```
Comment (ruthgdb) [inline Web/starter_project_vanguard/pages/about/index.tsx:50]: [nit] Is there a reason you're trying to change the font for this span only?

**Review by ruthgdb — CHANGES_REQUESTED** [review]

File: Web/starter_project_vanguard/tailwind.config.js [hunk 2]
```diff
@@ -11,21 +11,22 @@ module.exports = {
       colors: {
         primary: '#264FAD',
         accent: '#FF9F43',
+        aboutbackground: '#FDFEFF',
```
Comment (ruthgdb) [inline Web/starter_project_vanguard/tailwind.config.js:14]: [nit] the background color for most the pages is the same so can you give it a more inclusive name?

File: Web/starter_project_vanguard/tailwind.config.js [hunk 3]
```diff
@@ -11,21 +11,22 @@ module.exports = {
...
         danger: '#b91c1c',
         'primary-text': '#353535',
         'secondary-text': '#717171',
         'tertiary-text': '#565656',
-        
+        'aboutnormal-text': '#2B2A35',
```
Comment (ruthgdb) [inline Web/starter_project_vanguard/tailwind.config.js:19]: [nit] can you find a better name for this as well?

**Review by ruthgdb — CHANGES_REQUESTED** [review]

File: Web/starter_project_vanguard/components/about/A2SVSession.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import Image from 'next/image'
+
+function A2SVSession({ image, title, content }: any) {
```
Comment (sam5235) [inline Web/starter_project_vanguard/components/about/A2SVSession.tsx:3]: why the type is any, you can make it string.

File: Web/starter_project_vanguard/components/about/Activities.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import ImageTextStack from '@/components/about/ImageTextStack'
+import Image from 'next/image'
+import developmentImage from '../../public/images/aboutus/phases/development-phase.svg'
+import educationImage from '../../public/images/aboutus/phases/education-process.svg'
+import growthRateImage from '../../public/images/aboutus/phases/growth-rate.svg'
+
+function Activities({ studentGrowthRate, learningRate, growthRate }: any) {
```
Comment (sam5235) [inline Web/starter_project_vanguard/components/about/Activities.tsx:7]: remove all "any" types

File: Web/starter_project_vanguard/components/about/CenteredImage.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import Image from 'next/image'
+
+function CenteredImage({ image }: any) {
+  return (
+    <div className="mx-10 p-10 my-auto">
+      <Image src={image} alt={''}></Image>
```
Comment (sam5235) [inline Web/starter_project_vanguard/components/about/CenteredImage.tsx:6]: put something in the alt, this comment is for all images too

File: Web/starter_project_vanguard/pages/about/index.tsx [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+    up-and-coming developers use their skills to benefit Africa, rather
+    than taking their talent elsewhere."
+          />
+        </div>
+      </div>
+      <div className="grid grid-cols-2 gap-10">
+        <p className="font-extrabold text-secondary text-4xl my-10 col-span-2 text-center">
+          <span className="font-lato text-primary">Social </span> Projects
+        </p>
+        {socialProjects.map((project) => {
```
Comment (sam5235) [inline Web/starter_project_vanguard/pages/about/index.tsx:92]: every child should have key....this might cause vercel error.

**Review by sam5235 — APPROVED** [review]

File: Web/starter_project_vanguard/public/img/aboutus/icons/africa-icon.svg
Comment (ruthgdb) [inline Web/starter_project_vanguard/public/img/aboutus/icons/africa-icon.svg:1]: Can you rename the folder "aboutus" to "about" to make it more consistent

File: Web/starter_project_vanguard/components/about/A2SVSession.tsx [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import { SessionComponentProp } from '@/types/about/session-component-prop'
+import Image from 'next/image'
+
+function A2SVSession({ image, title, content }: SessionComponentProp) {
```
Comment (ruthgdb) [inline Web/starter_project_vanguard/components/about/A2SVSession.tsx:4]: Can you also add recat.FC to all your component files as well

**Review by ruthgdb — APPROVED** [review]
LGTM. Good job.

**Pull request merged**